pages/resources/jobExplorer/projects/[slug].tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import LandingNavbar from "../../../../components/landingPage/LandingNavbar";
+import JobExplorerProject from "../../../../components/resources/jobExplorer/JobExplorerProject";
+import {
+  getJobExplorerProjectData,
+  JobExplorerProjectData,
+} from "../../../api/resources/jobExplorer/getJobExplorerProjectData";
+
+type JobExplorerProjectPageProps = {
+  projectData: JobExplorerProjectData;
+};
+
+export default function JobExplorerProjectPage(
```

Comment:
this is the generic project page that we should be using for all projects going forward